magento/pwa-studio
# [PWA-480] Price summary does not get updated after removing an item.

## Description

URL - https://develop.pwa-venia.com/

Steps -
1. Add 2 products to cart and go to /cart page
2. Remove one product from cart page

Expected - Price summary should get updated for remaining products.
Actual - Price summary is blank, only labels no values.

## Related Issue





* [[PWA-480](https://jira.corp.magento.com/browse/PWA-480)] Price summary does not get updated after removing an item.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go through repro steps in description, verify bug no longer present
2. Step through other price adjustment panels that affects cart updating state (totals go away while mutations are in flight), verify they still work. 

## Screenshots / Screen Captures (if appropriate)
![](https://i.gyazo.com/b8878804994b7120f82f0eec528294de.gif)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
